Python/optimizer.c

```diff
@@ -857,6 +857,10 @@ uop_optimize(
     if (err < 0) {
         return -1;
     }
+    if (err == 1) {
+        return 0;
+    }
```

Comment:
This makes me feel there's some inconsistency in error return values that we could fix.